From 72a44c0c08923b8a0a51b02f8afd1943a47c6685 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 3 Nov 2022 17:27:01 +0100
Subject: [PATCH 232/954] feat(fdts): add default hconf1 OTP value for
 stm32mp257f-ev

Add the default shadow value to allow 1.8V management
on VDDIO1&2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git e1567ab3bd5a8770c04320e72db67dc58988f1fd feat(fdts): add default hconf1 OTP value for stm32mp257f-ev]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I798a36c178fdcc1709785c100f0b0c9793a3a956
---
 fdts/stm32mp257f-ev1.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index e481ee1f04..390510cff6 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -29,6 +29,15 @@
 		device_type = "memory";
 		reg = <0x0 0x80000000 0x1 0x0>;
 	};
+
+	shadow-prov {
+		compatible = "st,provisioning";
+
+		hconf1_prov {
+			nvmem-cells = <&hconf1_otp>;
+			st,shadow-value = <0x00018000>;
+		};
+	};
 };
 
 &usart2 {
-- 
2.53.0

